test/e2e/l7networkpolicy_test.go

```diff
@@ -216,7 +217,26 @@ func testL7NetworkPolicyHTTP(t *testing.T, data *TestData) {
 	require.NoError(t, NewPodBuilder(serverPodName, data.testNamespace, agnhostImage).OnNode(nodeName(0)).WithCommand(cmd).WithLabels(serverPodLabels).Create(data))
 	podIPs, err := data.podWaitForIPs(defaultTimeout, serverPodName, data.testNamespace)
 	require.NoError(t, err, "Expected IP for Pod '%s'", serverPodName)
-	serverIPs := podIPs.AsSlice()
+	dstPodIPs := podIPs.AsSlice()
+
+	// Create a Service whose backend is the above backend Pod.
+	var ipFamilies []corev1.IPFamily
+	if len(clusterInfo.podV4NetworkCIDR) != 0 {
+		ipFamilies = append(ipFamilies, corev1.IPv4Protocol)
+	}
+	if len(clusterInfo.podV6NetworkCIDR) != 0 {
+		ipFamilies = append(ipFamilies, corev1.IPv6Protocol)
+	}
+	mutator := func(service *corev1.Service) {
+		service.Spec.IPFamilies = ipFamilies
```

Comment:
would it be less verbose (and equivalent) to just set `.Spec.IPFamilyPolicy` to `PreferDualStack`?

Reply:
Done